src/main/java/frc/robot/Climb.java

```diff
@@ -0,0 +1,52 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj.DigitalInput;
+
+public class Climb {
+    
+
+    private final VictorSP winchMotor;
+
+    private final Solenoid winchLock;
+
+    private final VictorSP armMotor;
+
+    private DigitalInput armLimitSwitch;
+
+    private boolean currentWinchLockState;
+
+    public Climb(int winchMotorPort, int winchLockPort, int armMotorPort, armLimitSwitchPort) {
```

Comment:
Minor note: I would like a consistent curly brace format. The other classes use `function {`. Line 20 is perfect, for example.